performance-storage-service/pss_project/api/migrations/0013_change_creator_password copy.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.0.6 on 2020-07-22 06:22
```

Comment:
nit: did you just copy the auto generated migration python file? the file name looks interesting though

Reply:
oops haha